NEWS.md

```diff
@@ -11,6 +11,7 @@ for human readability.
 - The AMR routines for `P4estMesh` and `T8codeMesh` were changed to work on the product
   of the Jacobian and the conserved variables instead of the conserved variables only
   to make AMR fully conservative ([#2028]). This may change AMR results slightly.
+- Subcell (IDP) limiting is now officially supported and not marked as experimental anymore.
```

Comment:
```suggestion
- Subcell (IDP) limiting is now officially supported and not marked as experimental anymore (see `VolumeIntegralSubcellLimiting`).
```

Reply:
Done in [eb84655](https://github.com/trixi-framework/Trixi.jl/pull/2084/commits/eb84655eb7a84b4f8f0812220059da0c79cbc2ae)